web/source/dom/preProcessor.ts

```diff
@@ -262,7 +270,17 @@ namespace com.keyman.dom {
       /* I732 END - 13/03/2007 MCD: Swedish: End positional keyboard layout code */
       
       // Only reached if it's a mnemonic keyboard.
-      if(processor.swallowKeypress || processor.keyboardInterface.processKeystroke(keyman.util.physicalDevice, Levent.Ltarg, Levent)) {
+      
+      // This pathway won't have .processKeyEvent's FF keycode check, so we must do it here on
+      // .processKeystroke's behalf.
+      if(!keyman.isEmbedded && keyman.util.device.browser == 'firefox') {
```

Comment:
We need to check if this is still the case for Firefox -- does the hyphen still generate a different key code than on IE/Edge/Chrome/etc?

Reply:
I didn't test the hyphen yet, but I did test (and confirm) the `=` key, which is also part of the FF keymap.

Reply:
Hyphen works correctly, but doesn't need keymapping.  `;` is the other character mapped, and it still needs keymapping in Firefox - just checked.

Reply:
Good stuff; forgot it was `=` and `;` and not hyphen. Thanks for checking!